config/environments/development.rb

```diff
@@ -1,6 +1,17 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  config.action_mailer.default_url_options = { host: 'localhost:3000' }
+
+  ActionMailer::Base.smtp_settings = {
+  :user_name => 'ponytester123@gmail.com',
```

Comment:
I think its best practice to abstract the username as well as the password into ENV's